docs/features/techdocs/getting-started.md

```diff
@@ -10,73 +10,215 @@ to use TechDocs.
 If you haven't setup Backstage already, start
 [here](../../getting-started/index.md).
 
-## Installing TechDocs
+> If you used `npx @backstage/create-app`, TechDocs may already be present.
+>
+> You should skip to `Setting the Configuration` below.
```

Comment:
May already be present? We don't know? :)
Provide a link to Setting the Configuration.

Reply:
Good point 😄 I think we most certainly know that they TechDocs will be there when using the create-app CLI.
I used this language because Scaffolder has it too. https://backstage.io/docs/features/software-templates/installation#installing-backstageplugin-scaffolder

But clearly, this can be revisited. Thanks for raising this Anders. :)